src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1888,6 +1888,9 @@
         - name: --vm-sizes
           type: string
           short-summary: Comma-separated list of sizes. Must use VirtualMachines agent pool type.
+        - name: --undrainable-node-behavior
+          type: string
+          short-summary: Defines the behavior for undrainable nodes during upgrade. The value should be "Cordon" or "Schedule". The default value is "Schedule".
```

Comment:
please be consistant with other parameters and use first-person verb.
It is better to define this parameter with `arg_type=get_enum_type()` if it is enumeable.
```suggestion
          short-summary: Define the behavior for undrainable nodes during upgrade. The default value is "Schedule".
```